.github/workflows/test_definitions.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+# This uses workflows in a different repo, refer to:
+# https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
+
+# This runs sailbot workspace workflows and actions.
+# Runs the Tests workflow in sailbot_workspace
+   
+# Run condition
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 1:00am UTC
+    - cron: '0 1 * * 5'
+  
+jobs:
+# Calls our workflows in sailbot_workspace 
+# https://github.com/UBCSailbot/sailbot_workspace
+  call-workflow-1-in-sailbot:
```

Comment:
rename to `tests`

Reply:
Changed to
```
jobs:
# Calls tests in sailbot_workspace 
```

Reply:
I meant rename `call-workflow-1-in-sailbot` to `tests`

Reply:
It is annoying that my even though my comment was only for the last line, GitHub shows 4 lines, making it unclear which line I am referring to.

Reply:
Haha no problem. Changed ```callworkflow-1-in-sailbot``` to ```tests```
```yml
  tests:
        uses: UBCSailbot/sailbot_workspace/.github/workflows/test_definitions.yml@user/colinli02/11-Publish-ROS-package-actions_FINAL
        with:
            repository: ${{ github.event.repository.name }}
            disable-vcs: false
```